packages/storybook/src/stories/Fonts.tsx

```diff
@@ -1,5 +1,5 @@
-import { Fragment } from 'react'
 import { secondary } from '@littlespoon/theme/lib/fonts'
+import { Fragment } from 'react'
 
 interface Props {
```

Comment:
Do you need to update your interface given your change?

Reply:
Updated